### PR TITLE
Revert Service Line Changes

### PR DESCRIPTION
**Pull request by vaibhavkhanna08**

### CODE REVIEW

**Review by ravishanigarapu — APPROVED** [review]
looks ok to me

File: src/app/app-modules/registrar/registration/register-demographic-details/register-demographic-details.component.ts [hunk 1]
```diff
@@ -554,27 +549,24 @@ export class RegisterDemographicDetailsComponent implements OnInit, OnDestroy {
...
-        blockName: this.demographicsMaster.otherLoc.districtList[0].blockName,
+        blockID: this.demographicsMaster.otherLoc.blockID,
+        blockName: this.demographicsMaster.otherLoc.blockName,
       },
     ];
-    this.demographicDetailsForm.patchValue({
-      blockID: this.demographicsMaster.otherLoc.districtList[0].blockId,
-      blockName: this.demographicsMaster.otherLoc.districtList[0].blockName,
-    });
+    // this.demographicDetailsForm.patchValue({
```
Comment (drtechie) [inline src/app/app-modules/registrar/registration/register-demographic-details/register-demographic-details.component.ts:556]: @vaibhavkhanna08 as mentioned twice earlier, committing commented-out code is a bad practice.

**Comment by vaibhavkhanna08** [comment]
@drtechie  This was existing code which was already commented. By mistake I had uncommented that code and again I commented thats why it is showing like that I have commented this code.

**Comment by drtechie** [comment]
Okay, NP.

**Pull request merged**